en-US/ranking/criteria.md

```diff
@@ -2,78 +2,173 @@
+* If a song has been edited to have higher tempo, `(Sped Up Ver.)` marker must be used at the end of the title. If the title contains similar marker by default, it should be replaced with `(Sped Up Ver.)`.
+* If a song has been edited to have lower tempo, `(Slowed Down Ver.)` marker must be used at the end of the title. If the title contains similar marker by default, it should be replaced with `(Slowed Down Ver.)`.
+* Special unicode characters must be filtered to their nearest standard equivalent. Stars such as `★ ☆` should be substituted with an asterisk (`*`). Other special characters should be romanized or left out on a case-by-case basis.
+
+# Media
+
+#### Rules
 
-If you're looking for the process of getting a mapset ranked, see: [Quaver Ranking Process](/docs/Ranking/Process).
+* **The total file size of the mapset must not exceed 50 MB.**
 
-## Mandatory
+## Background
 
-**You must follow all the requirements listed in this section. Failing to meet all the requirements will not allow your map to receive ranked status.**
+#### Rules
 
-### Metadata
+* **Every map in the mapset must have a background image.** Using different background images for different maps is acceptable.
+* **The resolution of a background image must be at least 1280x720.**
+* **The file size of a background image must not exceed 4 MB.**
+* **The background must be of exceptionally high quality.** 
+* **the background must be appropriate for players of all ages and should not contain explicit content.**
 
-The following pertains to metadata that your maps contain such as song artists, titles, and tags.
+#### Guidelines
 
-* **The artist and title must be absolutely correct.**
-* **The artist and title must be romanized.** However, non-romanized characters are allowed to be used in source and tags.
-* **The artist, title and source must not repeat in tags.** However, romanizing or translating the source in tags is encouraged, in case it's relevant.
-* **Tags must contain the genre of the song.**
-* **Every difficulty must have a difficulty name.**
+* The background should show relevance to the song in some way. While mappers are allowed creative freedom when it comes to choosing backgrounds, completely unrelated background images that do not compliment the song in any way should generally not be used.
 
-### Media
+## Audio
 
-The following pertains to media files such as background images and audio files that are contained in your mapset.
+#### Rules
 
-* **The resolution of background images must be at least 1280x720** and of exceptionally high quality. The file size of a background image must not exceed 4 MB.
-* **The mapset must contain only one audio file.** Multiple song file mapsets are not eligible to be ranked.
+* **The mapset must contain only one audio file.**  Mapsets with multiple audio files are not eligible to be ranked.
```

Comment:
Some Quaver map which are ports of osu!mania were ranked in Quaver with hitsounds/keysounds alongside their files. 

I know Quaver is really weird with handling these audio, but they kind of work in gameplay (ie, it is still overriding the default skin hitsound). So, should we make an exception for these? Or should the mapper just clear everything related to hs/ks.